### PR TITLE
Rebase dev onto v3

### PR DESCRIPTION
**Pull request by konstantin-msft**
- Reconcile changes after rebasing.
- Update `package-lock.json` files.

### CODE REVIEW

**Comment by tnorling** [comment]
> * Add `@azure/node-token-validation` to lerna bootstrapping list.

IIRC we were planning to remove this from our tooling since we're not working on it anymore. cc. @jo-arroyo @peterzenz

**Comment by konstantin-msft** [comment]
> > * Add `@azure/node-token-validation` to lerna bootstrapping list.
> 
> IIRC we were planning to remove this from our tooling since we're not working on it anymore. cc. @jo-arroyo @peterzenz

Even though we are not working on it anymore we keep updating its' [package.lock](https://github.com/AzureAD/microsoft-authentication-library-for-js/commit/55d5d6fa47dc60a15db7e76c8fa9bf6245238428#diff-5eec38202d47285de950e2e712d2d56482064ff2b7c683e0fc4d22c05c80380eL76) every release.

**Comment by tnorling** [comment]
> Even though we are not working on it anymore we keep updating its' [package.lock](https://github.com/AzureAD/microsoft-authentication-library-for-js/commit/55d5d6fa47dc60a15db7e76c8fa9bf6245238428#diff-5eec38202d47285de950e2e712d2d56482064ff2b7c683e0fc4d22c05c80380eL76) every release.

Right, I'm saying we would like to stop doing that.

**Comment by konstantin-msft** [comment]
> > Even though we are not working on it anymore we keep updating its' [package.lock](https://github.com/AzureAD/microsoft-authentication-library-for-js/commit/55d5d6fa47dc60a15db7e76c8fa9bf6245238428#diff-5eec38202d47285de950e2e712d2d56482064ff2b7c683e0fc4d22c05c80380eL76) every release.
> 
> Right, I'm saying we would like to stop doing that.

@peterzenz @jo-arroyo In case we decide to not work on `node-token-validtion` module anymore I can create a separate PR to lock `msal-common` dep version for it.

**Review by peterzenz — APPROVED** [review]
Automation for the rebase, not providing a detailed review.  There are no callouts on areas that need additional conflict resolution or additional review.

**Review by tnorling — APPROVED** [review]

File: change/@azure-msal-browser-936e7906-4429-4deb-a6b5-6cee0ee45120.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (hectormmg) [inline change/@azure-msal-browser-936e7906-4429-4deb-a6b5-6cee0ee45120.json:1]: I don't think these change files should be included. They were deleted before the last v2 release went out as they were integrated into changelogs.

Comment (hectormmg) [reply change/@azure-msal-browser-936e7906-4429-4deb-a6b5-6cee0ee45120.json:1]: I realize this was the state of dev before the release, we may need to pull from dev again into msal-browser-v3 after merging this PR.

**Review by hectormmg — APPROVED** [review]

File: change/@azure-msal-browser-936e7906-4429-4deb-a6b5-6cee0ee45120.json (hunk 1 above)
Comment (konstantin-msft) [reply change/@azure-msal-browser-936e7906-4429-4deb-a6b5-6cee0ee45120.json:1]: They will be deleted automatically during next rebase.

**Comment by sp90** [comment]
The bot link to "offical npm release" does not include `-alpha.0` only links to `3.0.0` 

FYI

**Pull request merged**